modules/claims_api/app/controllers/claims_api/v2/veteran_identifier_controller.rb

```diff
@@ -5,18 +5,19 @@
 module ClaimsApi
   module V2
     class VeteranIdentifierController < ClaimsApi::V2::ApplicationController
-      # TODO: REMOVE BEFORE IMPLEMENTATION
-      skip_before_action :authenticate, only: %i[find]
-      ICN_FOR_TEST_USER = '1012667145V762142'
-      # TODO: REMOVE BEFORE IMPLEMENTATION
-
       def find
         raise ::Common::Exceptions::Unauthorized if request.headers['Authorization'].blank?
```

Comment:
Does this return a 401? Just making sure its a 401 and not a 403

Reply:
Oh dang it.  I should remove that completely right?  Because the usual auth logic is now back in place.

Reply:
Actually, is this still necessary? shouldnt `authenticate` handle this use case and the use case where the auth token is there but invalid?

Reply:
> Actually, is this still necessary? shouldnt `authenticate` handle this use case and the use case where the auth token is there but invalid?

Nope, it's not needed.
I just forgot to remove it.
[Removed here.](https://github.com/department-of-veterans-affairs/vets-api/pull/7265/commits/7328e7c37a8c4a6fe45649819419ccaf06020d30)